Code/Engine/RendererFoundation/Resources/Implementation/ProxyTexture.cpp

```diff
@@ -36,6 +36,12 @@ ezResult ezGALProxyTexture::DeInitPlatform(ezGALDevice* pDevice)
   return EZ_SUCCESS;
 }
 
+ezResult ezGALProxyTexture::ReplaceExisitingNativeObject(void* pExisitingNativeObject)
+{
+  //TODO
```

Comment:
assert not implemented ?

Reply:
@C-Core I have actually no idea what ezGALProxyTexture does and what I should put there.

Reply:
ezGALProxyTexture allows you to treat an arrayslice or cubemap face as a regular texture2D. This is useful if you want to e.g. render one face of a reflection probe but the pipeline does only support rendering into regular texture2D. I can't think of a use case why you want to replace the native object on such a texture so assert not implemented is fine for now.

Reply:
Thanks for the info! Will add not implemented assert.